lib/balena-lint.ts

```diff
@@ -1,3 +1,6 @@
+// Tell typescript-eslint that this is a single run
+process.env.TSESTREE_SINGLE_RUN = 'true';
```

Comment:
How about adding it to the `config/.eslintrc.js` like v, instead of using the env var?
```
	parserOptions: {
		allowAutomaticSingleRunInference: true,
		project: 'tsconfig.json',
		sourceType: 'module',
	},
```
See: https://github.com/balena-io-modules/node-balena-lint/blob/master/config/.eslintrc.js#L8-L11

Reply:
Because our runs won't be automatically detected since we don't use the eslint cli, check out https://github.com/typescript-eslint/typescript-eslint/blob/90baceea968c6e10917feec19bc063c8bb9919b6/packages/typescript-estree/src/parseSettings/inferSingleRun.ts#L37-L52